WordPress/Classes/ViewRelated/Aztec/ViewControllers/AztecPostViewController.swift

```diff
@@ -78,6 +78,42 @@ class AztecPostViewController: UIViewController, PostEditor {
         return textView
     }()
 
+    /// Count word or characters by set language
+    ///
+    fileprivate(set) lazy var wordOrCharacterCountLabel: UILabel = {
+        let label = UILabel()
+        label.textColor = Colors.wordCount
+        label.font = Fonts.regular
+        label.isUserInteractionEnabled = false
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.numberOfLines = 1
+        // swiftlint:disable:next inverse_text_alignment
+        label.textAlignment = .right
+        label.isHidden = true
+        return label
+    }()
+
+    /// If the language the user chose in settings is character based
+    /// The lable will show the number of characters.
+    /// If the language is not supported the label will stay hidden
+    /// otherwise, show word count.
+    ///
+
+    fileprivate(set) lazy var isCharacterBasedLanguge: Bool? = {
```

Comment:
Two things here: I think `Bool?` is rarely a useful return type for functions/properties. It's makes the code at call-site near impossible to understand without reading the implementation, but thankfully There's A Better Way!

I really like modeling problems like this with simple, throwaway Swift enums, something like this:
(pardon the naming, it's just an example, but my jetlagged brain can't come up with anything better at this hour, hopefully it gets the point across):
```swift
enum WordLabelDisplayMode {
    case wordBased
    case characterBased
    case hidden
}
```

It's much more descriptive and makes reading the caller site a breeze :)

But more importantly: I'm not sure whether this is the _right thing to do here_. I know nothing about japanese/chinese/korean/thai languages, so I'm not sure whether "character count" is a useful replacement metric for those languages. 

I'd rather us hide it entirely (which is what Web seems to be doing), than do something that's culturally insensitive or just doesn't make sense. Perhaps we could get some polyglots to chime in here whether that makes sense?

Reply:
🙏 Thank you for this comment! Turning this to an enum looks much better! 
For the character counting. I saw a comment on the Web project saying character count should be added. But I know consistency is really important so Im gonna remove this and just not show the count in those cases.